templates/block-php.mustache

```diff
@@ -24,31 +26,29 @@ function {{machine_name}}_enqueue_block_editor_assets() {
 	);
 
 	$editor_css = '{{slug}}/editor.css';
-	wp_enqueue_style(
+	wp_register_style(
 		'{{slug}}-block-editor',
 		plugins_url( $editor_css, __FILE__ ),
 		array(
 			'wp-blocks',
 		),
 		filemtime( "$dir/$editor_css" )
 	);
-}
-add_action( 'enqueue_block_editor_assets', '{{machine_name}}_enqueue_block_editor_assets' );
-
-/**
- * Enqueues block assets to apply both on the front of your site and in the editor.
- */
-function {{machine_name}}_enqueue_block_assets() {
-	$dir = dirname( __FILE__ );
 
 	$style_css = '{{slug}}/style.css';
-	wp_enqueue_style(
-        '{{slug}}-block',
+	wp_register_style(
+		'{{slug}}-block',
 		plugins_url( $style_css, __FILE__ ),
 		array(
-            'wp-blocks',
+			§'wp-blocks',
```

Comment:
You seem to have introduced an unintended change here: `§`

Reply:
I noticed that myself, too. It's already removed.